.github/workflows/run-benchmark.yml

```diff
@@ -85,12 +86,7 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           name: snakemake_results_linear-elastic-plate-with-hole
-          path: ./artifact_files
-
-      - name: Unzip metadata4ing_provenance.zip
-        run: |
-          mkdir -p ./metadata4ing_provenance
-          unzip -o ./artifact_files/metadata4ing_provenance.zip -d ./metadata4ing_provenance
+          path: ./snakemake_provenance
```

Comment:
the naming is not consistent, sometimes it is reporter_data, and then snakemake provenance.